packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -114,10 +115,11 @@ $iui-dialog-width: 380px;
   }
 }
 
-@mixin iui-dialog-draggable {
+@mixin iui-dialog-draggable($min-height: $iui-dialogue-resizable-min-height) {
```

Comment:
We only deliver the compiled CSS to our users, not the Sass mixins, so this parameter does nothing.

It can be a bit hard to follow inside the repo, so I suggest browsing the package in isolation here: https://unpkg.com/@itwin/itwinui-css/ This will give you an idea of what is "internal" vs "public".

Reply:
So is the only way to make it an optional variable that users can supply is through React?

Reply:
Through React props (for react users) or through CSS variables (for CSS users) or a combination of both in many cases. You can see an example in Badge:
https://github.com/iTwin/iTwinUI/blob/fd2e52397764d16ddd3132d9958817517016e086/packages/itwinui-css/src/badge/badge.scss#L21
https://github.com/iTwin/iTwinUI/blob/fd2e52397764d16ddd3132d9958817517016e086/packages/itwinui-react/src/core/Badge/Badge.tsx#L74

But for dialog min-height, I think there is no need to make it configurable. It should just be a sensible default value. Users always have the ability to override using their own custom CSS.